docs/serving/samples/hello-world/helloworld-java-spring/README.md

```diff
@@ -120,7 +119,7 @@ following commands:
    username.
 
     ```yaml
-    apiVersion: serving.knative.dev/v1alpha1
+    apiVersion: serving.knative.dev/v1beta1
```

Comment:
Since you sent this PR against release-0.6, we don't want to make this update to the beta API. For the master branch, this change has already been made: #1483